test/cpp/util/cli_credentials.cc

```diff
@@ -22,27 +22,43 @@
 
 DEFINE_bool(enable_ssl, false, "Whether to use ssl/tls.");
 DEFINE_bool(use_auth, false, "Whether to create default google credentials.");
+DEFINE_string(access_token, "",
+              "The access token in metadata \"authorization\".");
```

Comment:
metatada part is implementation details. How about saying something like "access token that will be sent to the server to authenticate the rpcs"..

Reply:
Done. Thanks for the suggestion.